tensorrt_llm/runtime/kv_cache_manager_v2/_core/_kv_cache.py

```
@@ -358,6 +359,12 @@ def __init__(
         )
         self._enable_request_stats = enable_request_stats
         self._pending_stats = _PendingStats()
+        self._reused_tokens_by_tier = {
```

Ruff (0.16.3)

[error] Attribute `_reused_tokens_by_tier` is not defined in class's `__slots__`

(PLE0237)